src/tools/android/java/com/google/devtools/build/android/ResourceProcessorBusyBox.java

```diff
@@ -177,24 +180,43 @@ public static void main(String[] args) throws Exception {
   }
 
   private static void runPersistentWorker() throws Exception {
-    while (true) {
-      try {
-        WorkRequest request = WorkRequest.parseDelimitedFrom(System.in);
 
-        if (request == null) {
-          break;
-        }
+    ByteArrayOutputStream buf = new ByteArrayOutputStream();
+    PrintStream ps = new PrintStream(buf);
```

Comment:
I think you'll want to enable autoFlush on 'ps' as well, to avoid including stdout chars in a stderr line and vice versa.

Reply:
@justhecuke PTAL

Reply:
Done.